### ISSUE TITLE
Хаб Тортуга

### ISSUE THREAD

**Issue by PapIK1122** [pavel-pimenov/flylinkdc-r5xx#1714]
После обновления появился какой-то хаб "Тортуга" 
![image](https://user-images.githubusercontent.com/34656047/45352275-ccda3180-b5c0-11e8-97ed-42fb215510e1.png)
![image](https://user-images.githubusercontent.com/34656047/45352287-d4013f80-b5c0-11e8-84a4-f67db8d7f212.png)
В списке избранных хабов его нету, в настройках тоже все излазил, никак не могу его отключить, что делать? 
Если закрыть, то при перезапуске программы он снова коннектиться.

**Comment by pavel-pimenov** [comment]
А руками можете добавить хаб в фавориты?
где у вас лежит Favorites.xml - может туда нет прав на запись?
включите системный лог в файл и покажите что там пишется - есть ошибки записи в Favorites.xml?

**Comment by PapIK1122** [comment]
Добавлял руками эту тортугу, убирал галочку, тоже не помогало.
![image](https://user-images.githubusercontent.com/34656047/45354397-2d1fa200-b5c6-11e8-92bd-2e51b4b431e8.png)
![image](https://user-images.githubusercontent.com/34656047/45354440-4a547080-b5c6-11e8-87b9-190164327ea6.png)
в файле тоже самое что и в программе
![image](https://user-images.githubusercontent.com/34656047/45354474-5fc99a80-b5c6-11e8-8363-74930024903e.png)
![image](https://user-images.githubusercontent.com/34656047/45354500-6e17b680-b5c6-11e8-845c-d65a3df27187.png)
Очень тупо, но после того как я открыл и закрыл этот файл, перезапускаю теперь флайлинг и он больше не появляется... Может потом появится...
[System.log](https://github.com/pavel-pimenov/flylinkdc-r5xx/files/2370611/System.log)

**Comment by pavel-pimenov** [comment]
System.log странный - он похож на лог от torrent-ов.
проблему повторить сможете? или закрываем вопрос?

**Comment by PapIK1122** [comment]
Закрываем. Хаб почему-то перестал появляться...
Если снова появится я напишу.
Спасибо.

**Comment by PapIK1122** [comment]
Короче я понял в чем проблема была. Я удалял его и не перезаходил. А после перезагрузки он снова появлялся. А когда я для Вас тестировал, я перезапустил программу и он больше появлялся. После обновления последнего еще какой-то новый хаб появился была та же проблема, я его удалял забывал перезапускать программу и он после выключения-включения компьютера снова появился, а потом вспомнил как я делал и действительно, если удалить хаб и перезайти в программу он больше беспокоить не будет. Пофиксите?

**Comment by pavel-pimenov** [comment]
вы выключаете комп по питанию? или корректно через винду?

**Comment by pavel-pimenov** [comment]
т.е. если вы меняете настройки в фаворитах - (например комментарий к хабу)
и выключаете комп - то оно тоже не сохраняется?

**Comment by pavel-pimenov** [comment]
Включите логирование в файл вот так:
![default](https://user-images.githubusercontent.com/920705/45829227-1e8e6480-bd03-11e8-8a8b-d4dadd618513.png)

и сделайте 
1. операцию с выключением.
2. операцию с закрытием программы.

и потом скиньте этот файл мне

**Comment by PapIK1122** [comment]
С описанием не проканало, изменил описание, не выходил из программы, потом выключил ПК, включил и все осталось. Завтра или сегодня, но позже простестирую с хабам каким, добавлю и буду пробовать избавиться
[System.log](https://github.com/pavel-pimenov/flylinkdc-r5xx/files/2406434/System.log)

**Comment by pavel-pimenov** [comment]
У вас system.log пишется не тот - покажите ваши настройки.
ваш лог это торрентовская часть

**Comment by PapIK1122** [comment]
![image](https://user-images.githubusercontent.com/34656047/45900793-6a1b3e00-bde9-11e8-92e3-b090425e4199.png)

**Comment by pavel-pimenov** [comment]
поставьте курсор на "Журнал системных сообщений"

**Comment by PapIK1122** [comment]
![image](https://user-images.githubusercontent.com/34656047/45900935-e4e45900-bde9-11e8-86ed-33196131657c.png)

**Comment by pavel-pimenov** [comment]
а теперь покажите выделив лог торрента - там какое имя файла стоит?
![default](https://user-images.githubusercontent.com/920705/45901011-1a894200-bdea-11e8-8cf2-04c7b248b23a.png)

**Comment by PapIK1122** [comment]
![image](https://user-images.githubusercontent.com/34656047/45901120-84095080-bdea-11e8-8e0a-3d11852f8c97.png)
Там галочки вообще нету)))

**Comment by pavel-pimenov** [comment]
это очень странно.
system.log должен выглядит так:
вы точно файл от туда берете?
![default](https://user-images.githubusercontent.com/920705/45901199-cfbbfa00-bdea-11e8-93c6-a908a1479fc1.png)

**Comment by PapIK1122** [comment]
![image](https://user-images.githubusercontent.com/34656047/45901362-57096d80-bdeb-11e8-8197-8c8cd19934f6.png)

**Comment by pavel-pimenov** [comment]
Отбой - в конце вашего файла нашел.
сотрите этот файл чтобы не было старого мусора

В нормальном режиме должна быть запись вида
[2018-09-20 18:32:55] MoveFile OK C:\Users\PapIK\AppData\Roaming\FlylinkDC++\Favorites.xml.tmp -> C:\Users\PapIK\AppData\Roaming\FlylinkDC++\Favorites.xml

получается если завершаете через винду - то
такой записи не делается.
и корректировки файла пропадают

**Comment by pavel-pimenov** [comment]
да. сегодня выйдет обновка. буду явно сохранять по событию выключения винды

**Comment by PapIK1122** [comment]
Отлично. Спасибо.

**Comment by pavel-pimenov** [comment]
Обновитесь и проверьте.

**Comment by PapIK1122** [comment]
Обновился. Подключился к официальному хабу поддержки, добавил его в избранные. Перезапустил программу. Он остался, все норм. Удалил его из избранных, остался только мой. Перезагрузился и
![image](https://user-images.githubusercontent.com/34656047/45916602-7f3baf80-be71-11e8-98f9-5615c5339ac2.png)
wtf что это новое появилось...

**Comment by PapIK1122** [comment]
[System.log](https://github.com/pavel-pimenov/flylinkdc-r5xx/files/2407717/System.log)

**Comment by PapIK1122** [comment]
Я кстати, его потом закрыл, снова перезагрузился, но он не пропал, в хабах его по прежнему нету. А вот когда я его закрыл снова и просто перезапустил программу - он пропал.

**Comment by pavel-pimenov** [comment]
это ты делал пока без перезапуска винда?
я ту часть вообще не трогал - оно и раньше так вероятнее всего работало.
давай с перезапускам винды разберемся сначала

**Comment by dcbeelinekz2** [comment]
ДА ВСЕ ЯСНО ТУТ
